GameLibrary/Database/DataContext.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Text.Json;
+
+namespace object_group_game.Database
+{
+	public class DataContext : DbContext
+	{
+		public DataContext()
+		{
+			//use when want to add tables
+			//Database.EnsureDeleted();
+			Database.EnsureCreated();
+		}
+
+		public DbSet<Character> Character { get; set; }
+		public DbSet<Location> Location { get; set; }
+		public DbSet<Item> Item { get; set; }
+		public DbSet<Effect> Effect { get; set; }
+		public DbSet<Player> Player { get; set; }
+
+		protected override void OnConfiguring (DbContextOptionsBuilder optionsBuilder)
+		{
+			// SQL server connection string
+			// Set user and password values appropriate to your server settings
+			optionsBuilder.UseMySQL(
+				"server=" + Configuration.GetInstance().Settings["server"]
+				+ ";database=" + Configuration.GetInstance().Settings["database"]
+				+ ";user=" + Configuration.GetInstance().Settings["user"]
+				+ ";password=" + Configuration.GetInstance().Settings["password"]
+				+ ";port=" + Configuration.GetInstance().Settings["port"]
+			);
+		}
+
+		protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
```

Comment:
Weird spacing

Reply:
Spacing problems might be due to GitHub, as they're not present on Editor, if the issues occur on your end, please change it accordingly